app/controllers/locations_controller.rb

```diff
@@ -0,0 +1,5 @@
+class LocationsController < ApplicationController
+  def index
+    @locations = Locations.near_postcode(params['postcode'])
```

Comment:
You love that `Array` coercion... My thought was that we'd add lower level tests to ensure an enumerable was return from that method once we know more about what it's meant to do